test/unit/solver/test_rosenbrock.cpp

```diff
@@ -96,14 +95,6 @@ void testAlphaMinusJacobian(std::size_t number_of_grid_cells)
   }
 }
 
-TEST(RosenbrockSolver, StandardAlphaMinusJacobian)
-{
-  testAlphaMinusJacobian<micm::Matrix, SparseMatrix, micm::LinearSolver<double, SparseMatrix>>(1);
-  testAlphaMinusJacobian<micm::Matrix, SparseMatrix, micm::LinearSolver<double, SparseMatrix>>(2);
-  testAlphaMinusJacobian<micm::Matrix, SparseMatrix, micm::LinearSolver<double, SparseMatrix>>(3);
-  testAlphaMinusJacobian<micm::Matrix, SparseMatrix, micm::LinearSolver<double, SparseMatrix>>(4);
-}
-
```

Comment:
Is there a reason that you remove these CPU tests?

Reply:
Thanks for catching these mistakes! It was not intentional to modify any testing file from the C++ side. I will add them on.

Reply:
I added it back.